### ISSUE TITLE
Interesting fix/hack/workaround for 'local' dependencies...

### ISSUE THREAD

**Issue by floooh** [floooh/fips#186]
Interesting idea by @code-disaster: https://gist.github.com/code-disaster/1b99511e0e74b21f97297e7091c56428

A 'real' local dependency mode should probably work in a project subdirectory not under version control (like NPM), but if the changes are small it may make sense to implement the idea in the gist just to enable more scenarios.

**Comment by code-disaster** [comment]
Remotely related corner case: transitive dependencies.

I just imported `fips-stb`. It has a dependency to `fips-glfw`, which is only needed to build the examples. Still, in "library mode", it also pulls the `glfw` dependency into the main project.

A current workaround is to just let the copy sit there (not a big deal), add `fips-glfw` to fips.yml inside the main project, then exclude it with e.g. `cond: "FIPS_NEVER"`.

**Comment by code-disaster** [comment]
Oh nvm, I've just seen `no_auto_import: true` in sokol-samples. Maybe this could be useful as a per-import flag.

**Comment by floooh** [comment]
Yep, it's a bit more work to setup no_auto_import projects, but it's created exactly for that reason, to filter out imports that are not needed in some build configs.
